apparmor.d/profiles-a-f/briar-desktop-tor

```diff
@@ -0,0 +1,67 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Roman Beslik <me@beroal.in.ua>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+profile briar-desktop-tor {
+  include <abstractions/base>
+
+  network inet stream,
+  network inet6 stream,
```

Comment:
It seems unlikely that inet6 was required: tor does not work over ipv6.

Reply:
Snowflake requires `network inet6 dgram`. I added `inet6` everywhere to make the profiles future-proof. My internet provider doesn't provide IPv6 actually.

Reply:
`briar-desktop` requires `network inet6 stream`.